api/external/applications/applications.proto

```diff
@@ -127,80 +279,106 @@ service ApplicationsService {
+		};
   };
 }
 
+
+// Request message for listing services.
 message ServicesReq {
+  // Applies search filters, in the format of `fieldname:value`.
   repeated string filter = 1;
+  // Applies pagination parameters.
   common.query.Pagination pagination = 2;
+  // Applies sorting parameters.
   common.query.Sorting sorting = 3;
 }
 
+// Request message for listing available service filters.
 message ServicesDistinctValuesReq {
+  // Field name of service values.
   string field_name = 1;
+  // Query value, supports wildcards (* and ?).
   string query_fragment = 2;
+  // Applies filters, in the format of `fieldname:value`.
   repeated string filter = 3;
 }
 
+// Request message for listing services by service group.
 message ServicesBySGReq {
+  // Service group ID.
   string service_group_id = 1;
+  // Applies pagination parameters.
   common.query.Pagination pagination = 2;
+  // Applies sorting parameters.
   common.query.Sorting sorting = 3;
+  // Applies filters, in the format of `fieldname:value`.
   repeated string filter = 5;
 }
 
+// Response message for GetServicesBySG.
 message ServicesBySGRes {
+  // Service group name.
   string group = 1;
+  // List of services.
   repeated Service services = 2;
+  // Intentionally blank.
   HealthCounts services_health_counts = 3;
 }
 
+// Response message for GetServices.
 message ServicesRes {
+  // List of services.
   repeated Service services = 1;
 }
 
+// Response message for GetServicesDistinctValues.
 message ServicesDistinctValuesRes {
+  // List of distinct values fitting query_fragment and filters.
   repeated string values = 1;
 }
 
+// Request message for GetDisconnectedServices.
 message DisconnectedServicesReq {
+  // Unused.
   reserved 1;
+  // Threshold for marking services disconnected in seconds.
   int32 threshold_seconds = 2;
 }
 
+// Request message for ServicesStats.
 message ServicesStatsReq {
 }
 
+// Response message for ServicesStats.
 message ServicesStatsRes {
+  // Total number of service groups reporting to Chef Automate.
   int32 total_service_groups = 1;
+  // Total number of services reporting to Chef Automate, counts both connected and disconnected services.
   int32 total_services = 2;
+  // Total number of supervisors reporting to Chef Automate.
   int32 total_supervisors = 3;
+  // Total number of deployments reporting to Chef Automate.
   int32 total_deployments = 4;
 }
 
-
+// Periodic job configuration.
 message PeriodicJobConfig {
+  // The job status? `false` is disabled, `true` is enabled.
   bool running = 1;
-  // To match the ingest API at /retention/nodes/missing-nodes/config, we use a
-  // string format that is a subset of elasticsearch's date math. See the
-  // simpledatemath package under lib/ for more details
+  // The `threshold` setting used by periodic jobs for evaluating services.
+  // Threshold is a string that follows Elasticsearch's date math expressions. For more information, see the simpledatemath package under `lib/`.
+
```

Comment:
```suggestion
```

Reply:
![Screen Shot 2019-12-10 at 5 15 32 PM](https://user-images.githubusercontent.com/22962/70582842-b7abd400-1b70-11ea-8dd9-a523129163e4.png)

This field is not getting a description, probably because of this extra blank line.

Reply:
Providing this description here would probably mean we don't need to describe threshold in every service that accepts it as a parameter.